features/admin.core.v1/components/modals/feature-preview-modal.tsx

```diff
@@ -55,6 +59,7 @@ import "./feature-preview-modal.scss";
 import { history } from "../../helpers/history";
 import { AppState } from "../../store";
 
+
```

Comment:
Remove this additional line

Reply:
Fixed with [b0973e3](https://github.com/wso2/identity-apps/pull/8511/commits/b0973e32850df79d6933131bf5d665db45e3f17e)